acquire/tools/decrypter.py

```diff
@@ -343,19 +343,17 @@ def main():
     if not progress:
         log.info("`rich` is not installed, progress will not be shown")
 
-    if args.output and len(args.files) > 1:
-        parser.exit("--output is only allowed when decrypting a single file")
+    if args.output and args.output.is_file() and len(args.files) > 1:
+        parser.exit("--output should be a directory when decrypting multiple files.")
 
-    if not args.output:
-        for path in args.files:
-            if path.suffix != ".enc":
-                parser.exit(f"File doesn't have .enc extension: {path}")
+    files: list[Path] = find_enc_files(args.files)
 
     if args.output:
-        outputs = [args.output.resolve()]
+        resolv_path = args.output.resolve()
+        outputs = [resolv_path / path.stem for path in files] if args.output.is_dir() else [resolv_path]
```

Comment:
`.stem` could lead to collisions if there are multiple input files with the same name but in different directories. But apart from recreating the whole sub-dir hierarchy of `path`, I don't see any other elegant options. Would it be worth to warn (or even error) if `len(set(ouputs)) != len(outputs)` or something like that?